eng/scripts/Language-Settings.ps1

```diff
@@ -7,9 +7,8 @@ $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/main/_data/rel
 $GithubUri = "https://github.com/Azure/azure-sdk-for-js"
 $PackageRepositoryUri = "https://www.npmjs.com/package"
 $ReducedDependencyLookup = @{
-  'core'       = @('@azure-rest/synapse-access-control', '@azure/arm-resources', '@azure/identity', '@azure/service-bus', '@azure/template')
-  'test-utils' = @('@azure-tests/perf-storage-blob', '@azure/arm-eventgrid', '@azure/ai-text-analytics', '@azure/identity', '@azure/template')
-  'identity'   = @('@azure-tests/perf-storage-blob', '@azure/ai-text-analytics', '@azure/arm-resources', '@azure/identity-cache-persistence', '@azure/identity-vscode', '@azure/storage-blob', '@azure/template')
+  'test-utils' = @('@azure-tests/perf-storage-blob')
```

Comment:
I assume these aren't in the trigger paths because these are artifacts in the ci.yml file. I wonder how much we care about these triggering those builds. It would be nice to eliminate this map and code to handle it completely. 

Also if these aren't in the ci.yml artifact lists then is there even a packageinfo for them and can they even be included in the package set?